contracts/plugins/assets/compoundv2/CTokenFiatCollateral.sol

```diff
@@ -37,9 +37,21 @@ contract CTokenFiatCollateral is AppreciatingFiatCollateral {
     /// Refresh exchange rates and update default status.
     /// @custom:interaction RCEI
     function refresh() public virtual override {
+        CollateralStatus oldStatus = status();
+
         // == Refresh ==
         // Update the Compound Protocol
-        cToken.exchangeRateCurrent();
+        // solhint-disable no-empty-blocks
+        try cToken.exchangeRateCurrent() {} catch (bytes memory errData) {
+            // see: docs/solidity-style.md#Catching-Empty-Data
+            if (errData.length == 0) revert(); // solhint-disable-line reason-string
+            markStatus(CollateralStatus.DISABLED);
+        }
+
+        CollateralStatus newStatus = status();
```

Comment:
Let's move lines 51-54 into the `catch` statement to save on gas. Only need to do the check if we executed the `markStatus(Collateral.DISABLED)`.